CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -2494,13 +2486,19 @@ local moduleApiTable = {}
 function moduleApiTable:Create(instanceType)
 	return function(data)
 		local obj = Instance.new(instanceType)
+		local parent = nil
 		for k, v in pairs(data) do
 			if type(k) == 'number' then
 				v.Parent = obj
+			elseif k == 'Parent' then
```

Comment:
This change is to ensure the Parent property is set last so the object is already initialized when parented to the DataModel, this can make creating the object faster in some cases.
More information here:
http://devforum.roblox.com/t/psa-dont-use-instance-new-with-parent-argument/30296